src/components/Records/Record/GeneralInfo.vue

```diff
@@ -38,16 +38,30 @@
         <div class="d-flex flex-column mt-2  ml-sm-6 ml-lg-8">
           <div class="d-flex flex-row mb-2 align-center">
             <h3>{{ currentRecord['fairsharingRecord'].name }}</h3>
-            <b class="ml-2">({{ currentRecord['fairsharingRecord'].abbreviation
-            }})</b>
+            <h3
```

Comment:
You can remove this line as name is a required field (there's always a name)

Reply:
![image](https://user-images.githubusercontent.com/25683981/90156738-b75d7b80-dd84-11ea-9b40-3762e9982ba2.png)
done